types/fabric/fabric-impl.d.ts

```diff
@@ -4480,3 +4481,50 @@ interface IUtil extends IUtilAnimation, IUtilArc, IObservable<IUtil>, IUtilDomEv
 	object: IUtilObject;
 	string: IUtilString;
 }
+
+export interface FilterBackend {
+	resources: Object;
+
+	applyFilters(filters: IBaseFilter[], sourceElement: HTMLImageElement | HTMLCanvasElement, sourceWidth: number, sourceHeight: number, targetCanvas: HTMLCanvasElement): any;
+
+	evictCachesForKey(cacheKey: any): void;
```

Comment:
Should `cacheKey` be [string](https://github.com/kangax/fabric.js/blob/1e5c64ff2ae1f7c32367f8a64616d9bd6919d425/src/shapes/image.class.js#L163)?